README.md

```diff
@@ -35,6 +35,21 @@ release a new version, update the version number in `version.rb`, and then run
 git commits and tags, and push the `.gem` file to
 [rubygems.org](https://rubygems.org).
 
+### Copying files to Docker Container
+
+The `RakeDocker.define_image_tasks()` in the `Rakefile` returns a list of 
+`Tasks`. To `COPY` files to a Docker container, configure the tasks by setting
```

Comment:
The fact that `RakeDocker.define_image_tasks` returns a list of tasks is independent from this feature. Actually, we are passing a block to that method. The method takes that block as an argument and can in turn pass it around or call it at a later point in time.